dipy/testing/decorators.py

```diff
@@ -73,3 +73,18 @@ def test_with_xvfb(*args, **kwargs):
     # Plant it back in and return the new function:
     test_with_xvfb.__name__ = fname
     return test_with_xvfb if not is_windows else my_test
+
+
+def set_random_seed(func):
+    """Decorator to set equal seeds.
+
+    This will help us avoid random errors
+    that occur with low probabilitiy during testing.
```

Comment:
The primary purpose of this is not to prevent errors, but to make results reproducible.